OPAL/br/src/main/scala/org/opalj/br/analyses/cg/EntryPointFinder.scala

```diff
@@ -347,4 +347,42 @@ object AllEntryPointsFinder extends EntryPointFinder {
             project.allProjectClassFiles.flatMap(_.methodsWithBody)
         else project.allMethodsWithBody
     }
+}
+
+/**
+ * The AndroidEntryPointFinder considers specific methods of app components as entry points.
+ *
+ * @author Tom Nikisch
+ */
+object AndroidEntryPointsFinder extends EntryPointFinder {
```

Comment:
Since you didn't answer: 
Are you planning to add support for the androidx/support library version of Activity etc? e.g. https://developer.android.com/reference/androidx/activity/package-summary or https://developer.android.com/reference/android/support/packages
If you're not doing that, please at least add a comment that makes this clear.

Reply:
Currently i am not planning to do that but i can probably do it later. Thank you for your suggestion.